helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java

```diff
@@ -87,7 +87,7 @@
   private static final Prober DEFAULT_PROBER = new DefaultProber();
   private static final String DEFAULT_LOCAL_HOST_FILTER = ".+";
   private static final String DEFAULT_PREFIX_DIRECTORY = "/tmp/helios-temp-jobs";
-  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "./helios-reports/test";
+  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "target/helios-reports/test";
```

Comment:
if you don't use maven, and consequently don't have the target directory, then the only thing that will happen is that we create the target directory and put stuff in there. is that really so bad?